dynamic-cat-api/function.json

```diff
@@ -9,8 +9,8 @@
         "get",
         "post"
       ],
-      "route": "cat/says/{text:alpha}"
-    },
+      "route": "cat/says/{text}"
```

Comment:
The alpha is from an example when the user selects from a list. By removing it, the value is assumed to be a string rather than an enum so can expect spaces. This fixes #4